src/app/services/sync/voyager.service.ts

```diff
@@ -78,7 +78,7 @@ export class VoyagerService {
     } else {
       options.openShiftConfig = this.openShift.getConfig();
     }
-    options.authContextProvider = this.auth.getAuthContextProvider();
+    options.authContextProvider = this.injector.get(AuthService).auth.getAuthContextProvider();
```

Comment:
The reason for this is to reduce the hard dependency on the Auth service so we don't create a circular dependency